kili/mutations/asset/helpers.py

```diff
@@ -67,7 +68,9 @@ def process_json_content(input_type: str,
     """
     if json_content_array is None:
         return [''] * len(content_array)
-    if input_type == 'FRAME':
+    if input_type in ('VIDEO', 'FRAME'):
+        if input_type == 'FRAME':
+            warnings.warn("FRAME input type is deprecated. Please use VIDEO instead")
```

Comment:
this warning will be raised every time that a client call append_many_to_dataset in the SDK. 
Is it very usefull ? because if he has created a Frame project in the past with the old input-type, he will not be able to change that to a 'VIDEO' input type by its own